bundle/rhpam/metadata/annotations.yaml

```diff
@@ -4,9 +4,9 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: rhpam-kogito-operator
```

Comment:
did you make the change ?

Reply:
@radtriste yes, but when I run the make bundle command it seems to be overridden but I cannot underderstand where is managed that annotation, I suspect in the Makefile package -> https://github.com/kiegroup/kogito-operator/blob/main/Makefile.rhpam#L66

Reply:
@radtriste @davidesalerno I'll be merging it, if needed we can revisit later. probably new changes will be needed in the coming days.